libnm-core/nm-setting-sriov.c

```diff
@@ -33,6 +33,7 @@
  */
 struct _NMSettingSriov {
 	NMSetting parent;
+	gboolean enable;
```

Comment:
could we reorder the fields so that they fill the gaps of the alignment? Basically, fields of the same type should follow each other. Also, inside a struct, it seems better to have `bool :1` bitfields. Not that it makes a difference in this case (in terms of size of the struct), but as a rule.

Reply:
ok